templates/common/points_indicator/component.html

```diff
@@ -0,0 +1,30 @@
+<div class="points-indicator--{{ result }}">
+	<svg class="icon">
+		{% if result == "not-started" %}
+			{% include "common/points_indicator/icons/not-started.svg" %}
+		{% elif result == "waiting" %}
+			{% include "common/points_indicator/icons/waiting.svg" %}
+		{% elif result == "points" %}
+			{% if score == max_points %}
+				{% include "common/points_indicator/icons/points-full.svg" with score=score max_points=max_points %}
+			{% else %}
+				{% include "common/points_indicator/icons/points.svg" with score=score max_points=max_points %}
```

Comment:
We could use term `points` in place of `score` as that is used in other locations of the code.

Reply:
Will do this, thanks